src/Navigation/styles.js

```diff
@@ -4,11 +4,11 @@ import styled from 'styled-components';
 /* Overall Navigation Component */
 
 export const NavigationWrapper = styled.div`
-    position: ${props => props.positionValue || 'static'};
-    top: ${props => props.topValue || 'auto'};
-    max-height: ${props => props.navMaxHeight || '80vh'};
-    width: ${props => props.navWidth || 'inherit'};
-    background-color: ${props => props.backgroundColor || 'inherit'};
+    position: static;
+    top: auto;
```

Comment:
`auto` is the default value for `top`, so this can be removed.

Reply:
Thank you for the feedback